tests/Testing/Console/PromptsValidationTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Illuminate\Tests\Testing\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Console\Kernel;
+use Orchestra\Testbench\TestCase;
+
+use function Laravel\Prompts\text;
+
+class PromptsValidationTest extends TestCase
```

Comment:
This test should be under `tests/Integration/Testing/Console` directory.

Reply:
thanks @crynobone, moved as per suggestion